test/test_von_mises.cpp

```diff
@@ -0,0 +1,379 @@
+// Copyright Paul A. Bristow 2010.
+// Copyright John Maddock 2007.
+
+// Use, modification and distribution are subject to the
+// Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+// test_von_mises.cpp
+
+// https://en.wikipedia.org/wiki/Von_Mises_distribution
+// From MathWorld--A Wolfram Web Resource.
+// http://mathworld.wolfram.com/VonMisesDistribution.html
+
+#include <pch.hpp> // include directory /libs/math/src/tr1/ is needed.
```

Comment:
Do you need the entire `pch.hpp`? If not I'd just include what you need.

Reply:
Not necessary at all.

Reply:
Appearently necessary for some architectures …